src/web/templates/edit_queue.html

```diff
@@ -1,6 +1,11 @@
 {% extends "layout.html" %}
 {% block content %}
-<div><h2>{{ queue.name|e }} (ID {{ queue.id }})</h2></div>
+<div style="margin: 5px">
+    <input type="text" id="new_queue_name" class="new_queue_name" size="30" placeholder="{{ queue.name|e }}">
```

Comment:
Why do we need both an ID and a class for this element?

Reply:
You are right, we dont.